pkg/nugetconfig/parse_test.go

```diff
@@ -0,0 +1,57 @@
+package nugetconfig
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/aquasecurity/go-dep-parser/pkg/types"
+)
+
+func TestParse(t *testing.T) {
+	tests := []struct {
+		name      string // Test input file
+		inputFile string
+		want      []types.Library
+		wantErr   string
+	}{
+		{
+			name:      "Config",
```

Comment:
Since, packages.config is legacy format, it is supported when creating .net project in windows with older .NET framework using Visual studio IDE. e.g. https://github.com/microsoft/AzureBot/blob/master/AzureBot/packages.config

Reply:
The document says `packages.config` is created when the first NuGet operation is run. Is it really impossible to generate it?

>If used, packages.config is typically located in a project root. It's automatically created when the first NuGet operation is run, but can also be created manually before running any commands such as nuget restore.

https://docs.microsoft.com/en-us/nuget/reference/packages-config

Reply:
Here's what I mean:
https://stackoverflow.com/questions/42293369/missing-packages-config-file
It is little tricky to create using new dotnet cli commands but can be added easily through Visual studio IDE with some tweaks to it's package manager settings.

Reply:
![image](https://user-images.githubusercontent.com/14909563/127777092-2aac6e4c-6cab-4c60-9819-0085f9180d38.png)